Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/FRFS/OnCancel.hs

```diff
@@ -95,6 +103,26 @@ onCancel _ booking' dOnCancel = do
           when (Just charges /= dOnCancel.cancellationCharges) $
             throwError $ InternalError "Cancellation Charges mismatch in onCancel Req"
         _ -> throwError $ InternalError "Refund Amount or Cancellation Charges not found in booking"
+    sendTicketCancelSMS :: Maybe Text -> Maybe Text -> Flow ()
+    sendTicketCancelSMS mRiderNumber mRiderMobileCountryCode =
+      whenJust booking'.partnerOrgId $ \pOrgId -> do
+        fork "send ticket cancel sms" $
+          withLogTag ("SMS:FRFSBookingId:" <> booking'.id.getId) $ do
+            mobileNumber <- mRiderNumber & fromMaybeM (PersonFieldNotPresent "mobileNumber")
+            smsCfg <- asks (.smsCfg)
+            let mocId = booking'.merchantOperatingCityId
+                countryCode = fromMaybe "+91" mRiderMobileCountryCode
+                phoneNumber = countryCode <> mobileNumber
+                sender = smsCfg.sender
```

Comment:
No need for this config, as sender will be present in `merchantMessage` and if you use the function I mentioned in the builder you will just have to pass the mobile number at end.